### DISCUSSION TITLE
update a existing sitemap

### DISCUSSION THREAD

**Discussion by jamalroger** [spatie/laravel-sitemap#399] [Q&A]
hi everyone , thanks a lot to this package
i am correclty i have sitemap and i want to update it add some url  but when read the documentation i did not found how.
i just want to know if already exist or we can add it.
thanks a lot.

**Comment by dinhkhanh** [comment]
+1 for this. 

I need to update an existing sitemap.

**Comment by joginder89** [comment]
It looks we can not add URL to existing sitemap.
